src/instructlab/model/backends/backends.py

```diff
@@ -329,12 +352,16 @@ def select_backend(cfg: serve_config) -> BackendServer:
         raise click.exceptions.Exit(1)
 
     host, port = split_hostport(cfg.host_port)
+    chat_template = cfg.chat_template
+    if not chat_template:
```

Comment:
I think this can be avoided by simply setting a default to chat_template (as suggested above).